indico/modules/events/papers/client/js/components/Paper.jsx

```diff
@@ -59,3 +108,77 @@ Paper.propTypes = {
   eventId: PropTypes.number.isRequired,
   contributionId: PropTypes.number.isRequired,
 };
+
+function PaperSteps({
+  isInFinalState,
+  isRejected,
+  hasEditable,
+  // eslint-disable-next-line no-shadow
+  editableURL,
+  fileTypes,
+  uploadableFiles,
+}) {
+  const {event, contribution} = useSelector(getPaperDetails);
+
+  return (
+    <Step.Group fluid>
+      <Step active={!isInFinalState} completed={isInFinalState}>
+        <Icon name="copy outline" />
+        <Step.Content>
+          <Step.Title>
+            <Translate>Peer Reviewing</Translate>
+          </Step.Title>
+          <Step.Description style={{marginBottom: 0}}>
+            {isInFinalState ? (
+              <Translate>The paper was reviewed</Translate>
+            ) : (
+              <Translate>Your paper is under review</Translate>
+            )}
+          </Step.Description>
+        </Step.Content>
+      </Step>
+      <Step active={isInFinalState} completed={hasEditable} disabled={isRejected}>
+        <Icon name="pencil alternate" />
+        <Step.Content>
+          <Step.Title>
+            {hasEditable || !isInFinalState || isRejected ? (
+              <Translate>Submit for Editing</Translate>
+            ) : (
+              <EditableSubmissionButton
+                eventId={event.id}
+                contributionId={contribution.id}
+                contributionCode={contribution.code}
+                fileTypes={{[EditableType.paper]: fileTypes}}
+                uploadableFiles={uploadableFiles}
+                text={Translate.string('Submit for editing')}
+              />
+            )}
+          </Step.Title>
+          {hasEditable && (
+            <Step.Description style={{marginBottom: 0}}>
+              <Translate>
+                The paper was{' '}
+                <Param name="submittedURL" value="submitted" wrapper={<a href={editableURL} />} />{' '}
```

Comment:
`name="url"` (no need to be overly verbose there).

Also, the hardcoded value kind of breaks i18n here. You need to use Param like this instead:

    <Param name="url" wrapper={<a href={editableURL} />}>submitted</Param>

Reply:
Thought I had moved the value within, maybe stashed it. Naming's good 👍